appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/ProjectToolbar.java

```diff
@@ -131,60 +130,32 @@ public void execute() {
         }
       });
     }
+  }
 
-    private boolean deleteConfirmation(List<Project> projects) {
-      String message;
-      GallerySettings gallerySettings = GalleryClient.getInstance().getGallerySettings();
-      if (projects.size() == 1) {
-        if (projects.get(0).isPublished()) {
-          message = MESSAGES.confirmDeleteSinglePublishedProjectWarning(projects.get(0).getProjectName());
-        } else {
-          message = MESSAGES.confirmMoveToTrashSingleProject(projects.get(0).getProjectName());
-        }
+  private boolean deleteConfirmation(List<Project> projects) {
```

Comment:
Why was this moved up in scope? It is only ever called from within `MoveToTrashAction`.